akka-http/src/main/scala/akka/http/scaladsl/server/directives/PathDirectives.scala

```diff
@@ -83,7 +83,7 @@ trait PathDirectives extends PathMatchers with ImplicitPathMatcherConstruction w
    * order, i.e. `pathSuffixTest("baz" / "bar")` would match `/foo/bar/baz`!
    */
   def pathSuffixTest[L](pm: PathMatcher[L]): Directive[L] = {
-    implicit def LIsTuple = pm.ev
+    implicit val LIsTuple = pm.ev
```

Comment:
Because it doesn't end up generating a method which reduces JAR size by a tiny bit... :)